Flow.Launcher/Languages/es-419.xaml

```diff
@@ -69,7 +70,7 @@
     <system:String x:Key="currentPriority">Prioridad Actual</system:String>
     <system:String x:Key="newPriority">Nueva Prioridad</system:String>
     <system:String x:Key="priority">Prioridad</system:String>
-    <system:String x:Key="priorityToolTip">Cambiar la prioridad del resultado del plugin</system:String>

```

Comment:
This shouldn't be deleted?

Reply:
The original string is updated, so i am not sure whether this is correct?

Reply:
Unless the translation has been updated in Crowdin, I think we should align these translation with this PR https://github.com/Flow-Launcher/Flow.Launcher/pull/1410/files

Reply:
ok